### PR TITLE
[Beta2.3][INFRASTRUCTURE] Use HA ports for the load balancers

### PR DESCRIPTION
**Pull request by KimForss**
## Problem
Firewall rules management is simplified using Ha Ports
https://azure.microsoft.com/en-us/updates/azure-load-balancer-standard-ha-ports/#:~:text=HA%20Ports%2C%20a%20premium%20offering%20of%20the%20Azure,required%20backend%20pool%2C%20regardless%20of%20the%20port%20numbers. 

## Solution
Change the Load Balancer Rules to use HA Ports

## Tests
<Please provide steps to test the PR>

## Notes
https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability

### CODE REVIEW

**Review by YunziZhangAzure — COMMENTED** [review]
There seems to be error:
https://dev.azure.com/azuresaphana/Azure-SAP-HANA/_build/results?buildId=16639&view=logs&j=13cfa893-be7e-5723-d322-b6d1bcec6aca&t=e3ab6a71-27fd-535d-bb6a-cc1428da20cd

Please check the test pipeline.

**Comment by KimForss** [comment]
> There seems to be error:
> https://dev.azure.com/azuresaphana/Azure-SAP-HANA/_build/results?buildId=16639&view=logs&j=13cfa893-be7e-5723-d322-b6d1bcec6aca&t=e3ab6a71-27fd-535d-bb6a-cc1428da20cd
> 
> Please check the test pipeline.

Fixed, the SKU type for the load balancer for anydb was incorrect

**Comment by YunziZhangAzure** [comment]
Pipeline fails.

**Comment by KimForss** [comment]
> Pipeline fails.

Now the pipeline succeeds (there is a change from the remote-tfstate2 to the new branch as the old branch created multiple load balancer rules whereas the new one only needs one)

**Comment by YunziZhangAzure** [comment]
LGTM!

File: deploy/terraform/terraform-units/modules/sap_system/hdb_node/infrastructure.tf [hunk 1]
```diff
@@ -71,13 +71,13 @@ resource "azurerm_network_interface_backend_address_pool_association" "hdb" {
...
 
 resource "azurerm_lb_rule" "hdb" {
-  count                          = local.enable_deployment ? length(local.loadbalancer_ports) : 0
+  count                          = local.enable_deployment ? 1 : 0
   resource_group_name            = var.resource_group[0].name
   loadbalancer_id                = azurerm_lb.hdb[0].id
-  name                           = format("%s%s%s%05d-%02d", local.prefix, var.naming.separator, local.resource_suffixes.db_alb_rule, local.loadbalancer_ports[count.index].port, count.index)
-  protocol                       = "Tcp"
-  frontend_port                  = local.loadbalancer_ports[count.index].port
-  backend_port                   = local.loadbalancer_ports[count.index].port
```
Comment (rukawata) [inline deploy/terraform/terraform-units/modules/sap_system/hdb_node/infrastructure.tf:80]: If local. loadbalancer_ports is no longer used, please remove them from variable tf.

Comment (KimForss) [reply deploy/terraform/terraform-units/modules/sap_system/hdb_node/infrastructure.tf:80]: done

**Review by rukawata — APPROVED** [review]

**Pull request merged**